scripts/in_container/check_environment.sh

```diff
@@ -99,12 +99,10 @@ function resetdb_if_requested() {
 
 function startairflow_if_requested() {
     if [[ ${START_AIRFLOW:="false"} == "true" ]]; then
+
         # initialize db and create the admin user if it's a new run
-        if $(airflow users list | grep 'Roles' -q);
-        then
-            airflow db init
-            airflow users create -u admin -p admin -f Thor -l Adminstra -r Admin -e dummy@dummy.email
-        fi
+        airflow db init
```

Comment:
Why do you think it's not needed ? I think where you use postgres and mysql backend, this is quite likely that the database is still running and the user is already created. I think this could be only removed for sqlite backend, but we should also support mysql/postgres

Reply:
You  can try this by running `./breeze --start-airflow --backend mysql` exitng and re-running it.

Reply:
it is not needed because if `db init` is executed again no data are lost and trying to create the Admin user again also will not raise and error (just the message that the user Admin is already there).

That said I spent some time trying to figure out the best way to check if the database was already initialized or not, just to discover that that check is already in the `db init` command :)

Reply:
Let me just double check and we can merge :)

Reply:
All good!

Reply:
Can you please rebase as @mik-laj asked ? This should fix the spelling problem

Reply:
done